apps/meteor/client/views/admin/cloud/hooks/useFeatureBullets.tsx

```diff
@@ -31,12 +31,6 @@ const useFeatureBullets = () => {
 				description: t('RegisterWorkspace_Features_Omnichannel_Description'),
```

Comment:
Would be very nice if you could replace the useTranslation from ui-context to `react-i18next`

Reply:
ok!